Cargo.toml

```diff
@@ -49,6 +50,7 @@ rustls-native-certs = "0.6"
 rustls-pemfile = "1"
 semver = "1"
 serde = { version = "1", features = ["derive"] }
+size_format = "1.0.2"
```

Comment:
should be sorted

Reply:
In case any misunderstanding, does the `sorted` mean alphabet? So I placed size_format behind serde. @nyurik

Reply:
exactly - the list of dependencies should be sorted alphabetically.  I have a shortcut in my IDE to select a few strings and sort them

Reply:
Hi @nyurik  Would you like to share it, the shortcut? Are you using IDEA or something else? I'm using VSCODE.

Reply:
you would need to create it for your vscode.  See https://stackoverflow.com/questions/69115477/how-can-i-order-alphabetically-a-list-in-plain-text-in-vs-code how to sort things, and https://stackoverflow.com/questions/33791097/how-can-i-change-keyboard-shortcut-bindings-in-visual-studio-code/33791170 on how to create shortcuts

Reply:
Sorted now.